e2e/src/app.po.ts

```diff
@@ -0,0 +1,12 @@
+import {browser, by, element} from 'protractor';
+
+export class MaterialDocsAppPage {
```

Comment:
Mark these functions `async`?

Reply:
I can do that, but it doesn't line up with what the Angular CLI generates.

Should we open an issue to update Angular CLI's e2e generation schematic to always make these `async` functions?

Reply:
It would be worth starting a discussion at least

Reply:
PR https://github.com/angular/angular-cli/pull/15785 proposes this change in `@schematics/angular`.

Reply:
It looks like using `async` functions and `await` breaks e2e tests on IE11, Edge, Safari, and Firefox. It also breaks on Chrome 41 and 75.